From: Petr Vorel <pvorel@suse.cz>
Date: Thu, 12 Feb 2026 15:40:04 +0100
Subject: [PATCH] scripts/spelling.txt: sort alphabetically

Easier to add new entries.  It was sorted when added in 66b47b4a9dad0, but
later got wrong order for few entries.  Sorted with en_US.UTF-8 locale.

Link: https://lkml.kernel.org/r/20260212144005.45052-1-pvorel@suse.cz
Signed-off-by: Petr Vorel <pvorel@suse.cz>
Cc: Jonathan Camerom <Jonathan.Cameron@huawei.com>
Cc: WangYuli <wangyuli@uniontech.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 scripts/spelling.txt | 330 +++++++++++++++++++++----------------------
 1 file changed, 165 insertions(+), 165 deletions(-)

diff --git a/scripts/spelling.txt b/scripts/spelling.txt
index e164d147f5418..84b05f57c1769 100644
--- a/scripts/spelling.txt
+++ b/scripts/spelling.txt
@@ -57,8 +57,8 @@ acknowledgement||acknowledgment
 ackowledge||acknowledge
 ackowledged||acknowledged
 acording||according
-activete||activate
 actived||activated
+activete||activate
 actualy||actually
 actvie||active
 acumulating||accumulating
@@ -66,12 +66,12 @@ acumulative||accumulative
 acumulator||accumulator
 acutally||actually
 adapater||adapter
+adddress||address
 adderted||asserted
 addional||additional
 additionaly||additionally
 additonal||additional
 addres||address
-adddress||address
 addreses||addresses
 addresss||address
 addrress||address
@@ -95,9 +95,9 @@ alegorical||allegorical
 algined||aligned
 algorith||algorithm
 algorithmical||algorithmically
+algorithmn||algorithm
 algoritm||algorithm
 algoritms||algorithms
-algorithmn||algorithm
 algorrithm||algorithm
 algorritm||algorithm
 aligment||alignment
@@ -128,20 +128,20 @@ amount of times||number of times
 amout||amount
 amplifer||amplifier
 amplifyer||amplifier
-an union||a union
-an user||a user
-an userspace||a userspace
-an one||a one
 analysator||analyzer
 ang||and
 anniversery||anniversary
 annoucement||announcement
 anomolies||anomalies
 anomoly||anomaly
+an one||a one
 anonynous||anonymous
+an union||a union
+an user||a user
+an userspace||a userspace
 anway||anyway
-aplication||application
 apeared||appeared
+aplication||application
 appearence||appearance
 applicaion||application
 appliction||application
@@ -155,8 +155,8 @@ approriately||appropriately
 apropriate||appropriate
 aquainted||acquainted
 aquired||acquired
-aquisition||acquisition
 aquires||acquires
+aquisition||acquisition
 arbitary||arbitrary
 architechture||architecture
 archtecture||architecture
@@ -189,30 +189,30 @@ assum||assume
 assumtpion||assumption
 asume||assume
 asuming||assuming
-asycronous||asynchronous
 asychronous||asynchronous
+asycronous||asynchronous
+asymetric||asymmetric
+asymmeric||asymmetric
 asynchnous||asynchronous
 asynchrnous||asynchronous
-asynchronus||asynchronous
 asynchromous||asynchronous
-asymetric||asymmetric
-asymmeric||asymmetric
+asynchronus||asynchronous
+atempt||attempt
 atleast||at least
 atomatically||automatically
 atomicly||atomically
-atempt||attempt
 atrributes||attributes
 attachement||attachment
 attatch||attach
 attched||attached
 attemp||attempt
-attemps||attempts
 attemping||attempting
+attemps||attempts
 attepmpt||attempt
 attnetion||attention
 attruibutes||attributes
-authentification||authentication
 authenicated||authenticated
+authentification||authentication
 automaticaly||automatically
 automaticly||automatically
 automatize||automate
@@ -288,19 +288,19 @@ calucate||calculate
 calulate||calculate
 cancelation||cancellation
 cancle||cancel
-cant||can't
-cant'||can't
-canot||cannot
-cann't||can't
 cannnot||cannot
+cann't||can't
+canot||cannot
+cant'||can't
+cant||can't
 capabiity||capability
 capabilites||capabilities
 capabilties||capabilities
 capabilty||capability
 capabitilies||capabilities
 capablity||capability
-capatibilities||capabilities
 capapbilities||capabilities
+capatibilities||capabilities
 captuer||capture
 caputure||capture
 carefuly||carefully
@@ -308,9 +308,9 @@ cariage||carriage
 casued||caused
 catagory||category
 cehck||check
+chache||cache
 challange||challenge
 challanges||challenges
-chache||cache
 chanell||channel
 changable||changeable
 chanined||chained
@@ -348,6 +348,7 @@ colescing||coalescing
 collapsable||collapsible
 colorfull||colorful
 comand||command
+comaptible||compatible
 comit||commit
 commerical||commercial
 comming||coming
@@ -358,10 +359,6 @@ committ||commit
 commmand||command
 commnunication||communication
 commoditiy||commodity
-comsume||consume
-comsumer||consumer
-comsuming||consuming
-comaptible||compatible
 compability||compatibility
 compaibility||compatibility
 comparsion||comparison
@@ -377,22 +374,25 @@ compleatly||completely
 completition||completion
 completly||completely
 complient||compliant
-componnents||components
 compoment||component
+componnents||components
 comppatible||compatible
 compres||compress
 compresion||compression
 compresser||compressor
 comression||compression
+comsume||consume
 comsumed||consumed
+comsumer||consumer
+comsuming||consuming
 comunicate||communicate
 comunication||communication
 conbination||combination
 concurent||concurrent
 conditionaly||conditionally
 conditon||condition
-condtion||condition
 condtional||conditional
+condtion||condition
 conected||connected
 conector||connector
 configed||configured
@@ -429,13 +429,13 @@ continous||continuous
 continously||continuously
 continueing||continuing
 contiuous||continuous
-contraints||constraints
-contruct||construct
 contol||control
 contoller||controller
+contraints||constraints
 controled||controlled
 controler||controller
 controll||control
+contruct||construct
 contruction||construction
 contry||country
 conuntry||country
@@ -466,10 +466,9 @@ debouce||debounce
 decendant||descendant
 decendants||descendants
 decompres||decompress
-decsribed||described
 decrese||decrease
 decription||description
-detault||default
+decsribed||described
 dectected||detected
 defailt||default
 deferal||deferral
@@ -483,9 +482,9 @@ defintion||definition
 defintions||definitions
 defualt||default
 defult||default
-deintializing||deinitializing
-deintialize||deinitialize
 deintialized||deinitialized
+deintialize||deinitialize
+deintializing||deinitializing
 deivce||device
 delared||declared
 delare||declare
@@ -495,8 +494,8 @@ delemiter||delimiter
 deley||delay
 delibrately||deliberately
 delievered||delivered
-demodualtor||demodulator
 demension||dimension
+demodualtor||demodulator
 dependancies||dependencies
 dependancy||dependency
 dependant||dependent
@@ -506,15 +505,15 @@ depreacte||deprecate
 desactivate||deactivate
 desciptor||descriptor
 desciptors||descriptors
-descritpor||descriptor
 descripto||descriptor
 descripton||description
 descrition||description
+descritpor||descriptor
 descritptor||descriptor
 desctiptor||descriptor
+desination||destination
 desriptor||descriptor
 desriptors||descriptors
-desination||destination
 destionation||destination
 destoried||destroyed
 destory||destroy
@@ -522,6 +521,7 @@ destoryed||destroyed
 destorys||destroys
 destroied||destroyed
 detabase||database
+detault||default
 deteced||detected
 detecion||detection
 detectt||detect
@@ -536,55 +536,54 @@ deveolpment||development
 devided||divided
 deviece||device
 devision||division
-diable||disable
 diabled||disabled
+diable||disable
 dicline||decline
+diconnected||disconnected
 dictionnary||dictionary
 didnt||didn't
 diferent||different
-differrence||difference
-diffrent||different
 differenciate||differentiate
+differrence||difference
 diffreential||differential
+diffrent||different
 diffrentiate||differentiate
 difinition||definition
 digial||digital
 dimention||dimension
 dimesions||dimensions
-diconnected||disconnected
-disabed||disabled
-disasembler||disassembler
-disble||disable
-disgest||digest
-disired||desired
-dispalying||displaying
-dissable||disable
-dissapeared||disappeared
 diplay||display
-directon||direction
 direcly||directly
+directon||direction
 direectly||directly
 diregard||disregard
-disassocation||disassociation
-disassocative||disassociative
+disabed||disabled
 disapear||disappear
 disapeared||disappeared
 disappared||disappeared
-disbale||disable
+disasembler||disassembler
+disassocation||disassociation
+disassocative||disassociative
 disbaled||disabled
-disble||disable
+disbale||disable
 disbled||disabled
+disble||disable
+disble||disable
 disconnet||disconnect
 discontinous||discontinuous
+disgest||digest
 disharge||discharge
+disired||desired
 disnabled||disabled
+dispalying||displaying
 dispertion||dispersion
+dissable||disable
+dissapeared||disappeared
 dissapears||disappears
 dissconect||disconnect
 distiction||distinction
 divisable||divisible
 divsiors||divisors
-dsiabled||disabled
 docuentation||documentation
 documantation||documentation
 documentaion||documentation
@@ -599,6 +598,7 @@ downlads||downloads
 droped||dropped
 droput||dropout
 druing||during
+dsiabled||disabled
 dyanmic||dynamic
 dynmaic||dynamic
 eanable||enable
@@ -622,20 +622,20 @@ enble||enable
 enchanced||enhanced
 encorporating||incorporating
 encrupted||encrypted
-encrypiton||encryption
 encryped||encrypted
+encrypiton||encryption
 encryptio||encryption
 endianess||endianness
-enpoint||endpoint
 enhaced||enhanced
 enlightnment||enlightenment
+enocded||encoded
+enought||enough
+enpoint||endpoint
 enqueing||enqueuing
+enterily||entirely
 entires||entries
 entites||entities
 entrys||entries
-enocded||encoded
-enought||enough
-enterily||entirely
 enviroiment||environment
 enviroment||environment
 environement||environment
@@ -654,8 +654,8 @@ evalute||evaluate
 evalutes||evaluates
 evalution||evaluation
 evaulated||evaluated
-excecutable||executable
 excceed||exceed
+excecutable||executable
 exceded||exceeded
 exceds||exceeds
 exceeed||exceed
@@ -669,41 +669,41 @@ exeuction||execution
 existance||existence
 existant||existent
 exixt||exist
-exsits||exists
 exlcude||exclude
 exlcuding||excluding
 exlcusive||exclusive
-exlusive||exclusive
 exlicitly||explicitly
+exlusive||exclusive
 exmaple||example
 expecially||especially
 experies||expires
 explicite||explicit
-explicity||explicitly
 explicitely||explicitly
-explict||explicit
+explicity||explicitly
 explictely||explicitly
+explict||explicit
 explictly||explicitly
 expresion||expression
 exprienced||experienced
 exprimental||experimental
-extened||extended
+exsits||exists
 exteneded||extended
+extened||extended
 extensability||extensibility
-extention||extension
 extenstion||extension
+extention||extension
 extracter||extractor
 faied||failed
 faield||failed
-faild||failed
 failded||failed
+faild||failed
 failer||failure
-faill||fail
 failied||failed
+faill||fail
 faillure||failure
+failng||failing
 failue||failure
 failuer||failure
-failng||failing
 faireness||fairness
 falied||failed
 faliure||failure
@@ -718,15 +718,15 @@ fetcing||fetching
 fileystem||filesystem
 fimrware||firmware
 fimware||firmware
+finanize||finalize
+findn||find
+finilizes||finalizes
+finsih||finish
 firmare||firmware
 firmaware||firmware
 firtly||firstly
 firware||firmware
 firwmare||firmware
-finanize||finalize
-findn||find
-finilizes||finalizes
-finsih||finish
 fliter||filter
 flusing||flushing
 folloing||following
@@ -743,9 +743,9 @@ forwared||forwarded
 frambuffer||framebuffer
 framming||framing
 framwork||framework
+frequancy||frequency
 frequence||frequency
 frequncy||frequency
-frequancy||frequency
 frome||from
 fronend||frontend
 fucntion||function
@@ -767,9 +767,9 @@ gatable||gateable
 gateing||gating
 gauage||gauge
 gaurenteed||guaranteed
-generiously||generously
 genereate||generate
 genereted||generated
+generiously||generously
 genric||generic
 gerenal||general
 geting||getting
@@ -791,18 +791,17 @@ hanlde||handle
 hanled||handled
 happend||happened
 hardare||hardware
-harware||hardware
 hardward||hardware
+harware||hardware
 havind||having
+hearbeat||heartbeat
 heigth||height
+heirachy||hierarchy
 heirarchically||hierarchically
 heirarchy||hierarchy
-heirachy||hierarchy
 helpfull||helpful
-hearbeat||heartbeat
 heterogenous||heterogeneous
 hexdecimal||hexadecimal
-hybernate||hibernate
 hiearchy||hierarchy
 hierachy||hierarchy
 hierarchie||hierarchy
@@ -810,14 +809,14 @@ homogenous||homogeneous
 horizental||horizontal
 howver||however
 hsould||should
+hybernate||hibernate
 hypervior||hypervisor
 hypter||hyper
 idel||idle
 identidier||identifier
 iligal||illegal
-illigal||illegal
 illgal||illegal
-iomaped||iomapped
+illigal||illegal
 imblance||imbalance
 immeadiately||immediately
 immedaite||immediate
@@ -832,13 +831,14 @@ implemantation||implementation
 implemenation||implementation
 implementaiton||implementation
 implementated||implemented
-implemention||implementation
 implementd||implemented
+implemention||implementation
 implemetation||implementation
 implemntation||implementation
 implentation||implementation
 implmentation||implementation
 implmenting||implementing
+inavlid||invalid
 incative||inactive
 incomming||incoming
 incompaitiblity||incompatibility
@@ -870,9 +870,9 @@ infromation||information
 ingore||ignore
 inheritence||inheritance
 inital||initial
-initalized||initialized
 initalised||initialized
 initalise||initialize
+initalized||initialized
 initalize||initialize
 initation||initiation
 initators||initiators
@@ -880,20 +880,20 @@ initialiazation||initialization
 initializationg||initialization
 initializiation||initialization
 initializtion||initialization
-initialze||initialize
 initialzed||initialized
+initialze||initialize
 initialzing||initializing
 initilization||initialization
+initilized||initialized
 initilize||initialize
 initliaze||initialize
-initilized||initialized
 inofficial||unofficial
 inrerface||interface
 insititute||institute
 instace||instance
 instal||install
-instanciate||instantiate
 instanciated||instantiated
+instanciate||instantiate
 instuments||instruments
 insufficent||insufficient
 intead||instead
@@ -912,16 +912,16 @@ intergrated||integrated
 intermittant||intermittent
 internel||internal
 interoprability||interoperability
-interuupt||interrupt
-interupt||interrupt
-interupts||interrupts
-interurpt||interrupt
 interrface||interface
 interrrupt||interrupt
 interrup||interrupt
 interrups||interrupts
 interruptted||interrupted
 interupted||interrupted
+interupt||interrupt
+interupts||interrupts
+interurpt||interrupt
+interuupt||interrupt
 intiailized||initialized
 intial||initial
 intialisation||initialisation
@@ -935,18 +935,18 @@ intrerrupt||interrupt
 intrrupt||interrupt
 intterrupt||interrupt
 intuative||intuitive
-inavlid||invalid
 invaid||invalid
 invaild||invalid
 invailid||invalid
-invald||invalid
 invalde||invalid
+invald||invalid
 invalide||invalid
 invalidiate||invalidate
 invalud||invalid
 invididual||individual
 invokation||invocation
 invokations||invocations
+iomaped||iomapped
 ireelevant||irrelevant
 irrelevent||irrelevant
 isnt||isn't
@@ -992,11 +992,11 @@ losted||lost
 maangement||management
 machinary||machinery
 maibox||mailbox
+mailformed||malformed
 maintainance||maintenance
 maintainence||maintenance
 maintan||maintain
 makeing||making
-mailformed||malformed
 malplaced||misplaced
 malplace||misplace
 managable||manageable
@@ -1006,21 +1006,22 @@ mangement||management
 manger||manager
 manoeuvering||maneuvering
 manufaucturing||manufacturing
-mappping||mapping
 maping||mapping
+mappping||mapping
 matchs||matches
 mathimatical||mathematical
 mathimatic||mathematic
 mathimatics||mathematics
-maxmium||maximum
 maximium||maximum
 maxium||maximum
+maxmium||maximum
 mechamism||mechanism
 mechanim||mechanism
 meetign||meeting
 memeory||memory
 memmber||member
 memoery||memory
+memomry||memory
 memroy||memory
 ment||meant
 mergable||mergeable
@@ -1037,19 +1038,19 @@ migrateable||migratable
 miliseconds||milliseconds
 millenium||millennium
 milliseonds||milliseconds
-minimim||minimum
-minium||minimum
 minimam||minimum
+minimim||minimum
 minimun||minimum
+minium||minimum
 miniumum||minimum
 minumum||minimum
 misalinged||misaligned
 miscelleneous||miscellaneous
 misformed||malformed
-mispelled||misspelled
-mispelt||misspelt
 mising||missing
 mismactch||mismatch
+mispelled||misspelled
+mispelt||misspelt
 missign||missing
 missmanaged||mismanaged
 missmatch||mismatch
@@ -1062,18 +1063,17 @@ modifer||modifier
 modul||module
 modulues||modules
 momery||memory
-memomry||memory
 monitring||monitoring
 monochorome||monochrome
 monochromo||monochrome
 monocrome||monochrome
 mopdule||module
 mroe||more
-mulitplied||multiplied
 muliple||multiple
-multipler||multiplier
+mulitplied||multiplied
 multidimensionnal||multidimensional
 multipe||multiple
+multipler||multiplier
 multple||multiple
 mumber||number
 muticast||multicast
@@ -1095,6 +1095,7 @@ nerver||never
 nescessary||necessary
 nessessary||necessary
 none existent||non-existent
+notfify||notify
 noticable||noticeable
 notication||notification
 notications||notifications
@@ -1102,7 +1103,6 @@ notifcations||notifications
 notifed||notified
 notifer||notifier
 notity||notify
-notfify||notify
 nubmer||number
 numebr||number
 numer||number
@@ -1120,10 +1120,10 @@ occurence||occurrence
 occure||occurred
 occuring||occurring
 ocurrence||occurrence
-offser||offset
 offet||offset
 offlaod||offload
 offloded||offloaded
+offser||offset
 offseting||offsetting
 oflload||offload
 omited||omitted
@@ -1142,25 +1142,25 @@ optionnal||optional
 optmizations||optimizations
 orientatied||orientated
 orientied||oriented
-orignal||original
 originial||original
+orignal||original
 orphanded||orphaned
 otherise||otherwise
 ouput||output
 oustanding||outstanding
+oveflow||overflow
 overaall||overall
+overflw||overflow
 overhread||overhead
+overide||override
 overlaping||overlapping
-oveflow||overflow
-overflw||overflow
 overlfow||overflow
-overide||override
 overrided||overridden
 overriden||overridden
 overrrun||overrun
 overun||overrun
-overwritting||overwriting
 overwriten||overwritten
+overwritting||overwriting
 pacakge||package
 pachage||package
 packacge||package
@@ -1170,11 +1170,11 @@ packtes||packets
 pakage||package
 paket||packet
 pallette||palette
-paln||plan
 palne||plane
+paln||plan
 paramameters||parameters
-paramaters||parameters
 paramater||parameter
+paramaters||parameters
 paramenters||parameters
 parametes||parameters
 parametised||parametrised
@@ -1242,8 +1242,6 @@ prefered||preferred
 prefferably||preferably
 prefitler||prefilter
 preform||perform
-previleged||privileged
-previlege||privilege
 premption||preemption
 prepaired||prepared
 prepate||prepare
@@ -1251,6 +1249,8 @@ preperation||preparation
 preprare||prepare
 pressre||pressure
 presuambly||presumably
+previleged||privileged
+previlege||privilege
 previosuly||previously
 previsously||previously
 primative||primitive
@@ -1259,17 +1259,17 @@ priorty||priority
 priting||printing
 privilaged||privileged
 privilage||privilege
-priviledge||privilege
 priviledged||privileged
+priviledge||privilege
 priviledges||privileges
 privleges||privileges
-probaly||probably
 probabalistic||probabilistic
+probaly||probably
 procceed||proceed
 proccesors||processors
 procesed||processed
-proces||process
 procesing||processing
+proces||process
 processessing||processing
 processess||processes
 processpr||processor
@@ -1289,6 +1289,7 @@ progresss||progress
 prohibitted||prohibited
 prohibitting||prohibiting
 promiscous||promiscuous
+promixity||proximity
 promps||prompts
 pronnounced||pronounced
 prononciation||pronunciation
@@ -1297,15 +1298,14 @@ pronunce||pronounce
 propery||property
 propigate||propagate
 propigation||propagation
-propogation||propagation
 propogate||propagate
+propogation||propagation
 prosess||process
 protable||portable
 protcol||protocol
 protecion||protection
 protedcted||protected
 protocoll||protocol
-promixity||proximity
 psudo||pseudo
 psuedo||pseudo
 psychadelic||psychedelic
@@ -1334,8 +1334,8 @@ recieves||receives
 recieving||receiving
 recogniced||recognised
 recognizeable||recognizable
-recompte||recompute
 recommanded||recommended
+recompte||recompute
 recyle||recycle
 redect||reject
 redircet||redirect
@@ -1345,8 +1345,8 @@ reename||rename
 refcounf||refcount
 refence||reference
 refered||referred
-referencce||reference
 referenace||reference
+referencce||reference
 refererence||reference
 refering||referring
 refernces||references
@@ -1354,8 +1354,8 @@ refernnce||reference
 refrence||reference
 regiser||register
 registed||registered
-registerd||registered
 registeration||registration
+registerd||registered
 registeresd||registered
 registerred||registered
 registes||registers
@@ -1372,8 +1372,8 @@ reloade||reload
 remoote||remote
 remore||remote
 removeable||removable
-repective||respective
 repectively||respectively
+repective||respective
 replacable||replaceable
 replacments||replacements
 replys||replies
@@ -1390,8 +1390,8 @@ requieres||requires
 requirment||requirement
 requred||required
 requried||required
-requst||request
 requsted||requested
+requst||request
 reregisteration||reregistration
 reseting||resetting
 reseved||reserved
@@ -1413,11 +1413,11 @@ retransmited||retransmitted
 retreived||retrieved
 retreive||retrieve
 retreiving||retrieving
-retrive||retrieve
 retrived||retrieved
+retrive||retrieve
 retrun||return
-retun||return
 retuned||returned
+retun||return
 reudce||reduce
 reuest||request
 reuqest||request
@@ -1465,9 +1465,9 @@ seperate||separate
 seperatly||separately
 seperator||separator
 sepperate||separate
-seqeunce||sequence
-seqeuncer||sequencer
 seqeuencer||sequencer
+seqeuncer||sequencer
+seqeunce||sequence
 sequece||sequence
 sequemce||sequence
 sequencial||sequential
@@ -1506,8 +1506,8 @@ soley||solely
 soluation||solution
 souce||source
 speach||speech
-specfic||specific
 specfication||specification
+specfic||specific
 specfield||specified
 speciefied||specified
 specifc||specific
@@ -1516,8 +1516,8 @@ specificatin||specification
 specificaton||specification
 specificed||specified
 specifing||specifying
-specifiy||specify
 specifiying||specifying
+specifiy||specify
 speficied||specified
 speicify||specify
 speling||spelling
@@ -1544,23 +1544,23 @@ stoppped||stopped
 straming||streaming
 struc||struct
 structres||structures
-stuct||struct
 strucuture||structure
+stuct||struct
 stucture||structure
 sturcture||structure
 subdirectoires||subdirectories
 suble||subtle
-substract||subtract
 submited||submitted
 submition||submission
+substract||subtract
 succeded||succeeded
-suceed||succeed
-succesfuly||successfully
 succesfully||successfully
 succesful||successful
+succesfuly||successfully
 successed||succeeded
 successfull||successful
 successfuly||successfully
+suceed||succeed
 sucessfully||successfully
 sucessful||successful
 sucess||success
@@ -1569,9 +1569,9 @@ superseeded||superseded
 suplied||supplied
 suported||supported
 suport||support
-supportet||supported
 suppored||supported
 supporing||supporting
+supportet||supported
 supportin||supporting
 suppoted||supported
 suppported||supported
@@ -1582,27 +1582,27 @@ surpressed||suppressed
 surpresses||suppresses
 susbsystem||subsystem
 suspeneded||suspended
-suspsend||suspend
 suspicously||suspiciously
+suspsend||suspend
 swaping||swapping
 switchs||switches
-swith||switch
 swithable||switchable
-swithc||switch
 swithced||switched
 swithcing||switching
+swithc||switch
 swithed||switched
 swithing||switching
+swith||switch
 swtich||switch
+sychronization||synchronization
+sychronously||synchronously
 syfs||sysfs
 symetric||symmetric
 synax||syntax
 synchonized||synchronized
-sychronization||synchronization
-sychronously||synchronously
 synchronuously||synchronously
-syncronize||synchronize
 syncronized||synchronized
+syncronize||synchronize
 syncronizing||synchronizing
 syncronus||synchronous
 syste||system
@@ -1611,16 +1611,17 @@ sythesis||synthesis
 tagert||target
 taht||that
 tained||tainted
-tarffic||traffic
+tansition||transition
 tansmit||transmit
+tarffic||traffic
 targetted||targeted
 targetting||targeting
 taskelt||tasklet
 teh||the
 temeprature||temperature
 temorary||temporary
-temproarily||temporarily
 temperture||temperature
+temproarily||temporarily
 theads||threads
 therfore||therefore
 thier||their
@@ -1630,23 +1631,20 @@ threshhold||threshold
 thresold||threshold
 throtting||throttling
 throught||through
-tansition||transition
-trackling||tracking
-troughput||throughput
-trys||tries
 thses||these
-tiggers||triggers
 tiggered||triggered
 tiggerring||triggering
-tipically||typically
+tiggers||triggers
 timeing||timing
 timming||timing
 timout||timeout
+tipically||typically
 tmis||this
 tolarance||tolerance
 toogle||toggle
 torerable||tolerable
 torlence||tolerance
+trackling||tracking
 traget||target
 traking||tracking
 tramsmitted||transmitted
@@ -1670,20 +1668,20 @@ trasfer||transfer
 trasmission||transmission
 trasmitter||transmitter
 treshold||threshold
-trigged||triggered
-triggerd||triggered
 trigerred||triggered
 trigerring||triggering
+trigged||triggered
+triggerd||triggered
+troughput||throughput
 trun||turn
+trys||tries
 tunning||tuning
 ture||true
 tyep||type
 udpate||update
-updtes||updates
 uesd||used
-unknwon||unknown
 uknown||unknown
-usccess||success
+unamed||unnamed
 uncommited||uncommitted
 uncompatible||incompatible
 uncomressed||uncompressed
@@ -1692,6 +1690,7 @@ undeflow||underflow
 undelying||underlying
 underun||underrun
 unecessary||unnecessary
+uneeded||unneeded
 unexecpted||unexpected
 unexepected||unexpected
 unexpcted||unexpected
@@ -1700,26 +1699,24 @@ unexpeted||unexpected
 unexpexted||unexpected
 unfortunatelly||unfortunately
 unifiy||unify
-uniterrupted||uninterrupted
 uninterruptable||uninterruptible
 unintialized||uninitialized
+uniterrupted||uninterrupted
 unitialized||uninitialized
 unkmown||unknown
 unknonw||unknown
 unknouwn||unknown
 unknow||unknown
+unknwon||unknown
 unkown||unknown
-unamed||unnamed
-uneeded||unneeded
-unneded||unneeded
+unmached||unmatched
 unneccecary||unnecessary
 unneccesary||unnecessary
 unneccessary||unnecessary
 unnecesary||unnecessary
+unneded||unneeded
 unneedingly||unnecessarily
 unnsupported||unsupported
-unuspported||unsupported
-unmached||unmatched
 unprecise||imprecise
 unpriviledged||unprivileged
 unpriviliged||unprivileged
@@ -1727,18 +1724,21 @@ unregester||unregister
 unresgister||unregister
 unrgesiter||unregister
 unsinged||unsigned
-unstabel||unstable
-unsolicted||unsolicited
 unsolicitied||unsolicited
+unsolicted||unsolicited
+unstabel||unstable
 unsuccessfull||unsuccessful
 unsuported||unsupported
 untill||until
 ununsed||unused
 unuseful||useless
+unuspported||unsupported
 unvalid||invalid
 upate||update
+updtes||updates
 upsupported||unsupported
 upto||up to
+usccess||success
 useable||usable
 usefule||useful
 usefull||useful
@@ -1760,14 +1760,14 @@ variantions||variations
 varible||variable
 varient||variant
 vaule||value
-verbse||verbose
 veify||verify
+verbse||verbose
 verfication||verification
 veriosn||version
-versoin||version
 verisons||versions
 verison||version
 veritical||vertical
+versoin||version
 verson||version
 vicefersa||vice-versa
 virtal||virtual
@@ -1780,13 +1780,13 @@ wakeus||wakeups
 was't||wasn't
 wathdog||watchdog
 wating||waiting
-wiat||wait
 wether||whether
 whataver||whatever
 whcih||which
 whenver||whenever
 wheter||whether
 whe||when
+wiat||wait
 wierd||weird
 wihout||without
 wiil||will
-- 
2.47.3

